### PR TITLE
Updated .json and .md files as per latest code

### PR DESCRIPTION
**Pull request by RKarthikeyanRaja**
Documentation enhancement for HdmiInput plugin
JIRA ID: https://ccp.sys.comcast.net/browse/RDK-34902

### CODE REVIEW

**Review by ddevad — APPROVED** [review]

File: HdmiInput/doc/HdmiInputPlugin.md [hunk 1]
```diff
@@ -333,18 +361,26 @@ Returns the current EDID value.
...
 <a name="method.startHdmiInput"></a>
-## *startHdmiInput <sup>method</sup>*
+## *startHdmiInput [<sup>method</sup>](#head.Methods)*
 
 Activates the specified HDMI Input port as the primary video source.
+ 
+### Events 
+| Event | Description | 
+| :----------- | :----------- | 
+| `onInputStatusChanged` | Triggers the event when activates HDMI Input source and Input status changes to `started` |
```
Comment (anand-ky) [inline HdmiInput/doc/HdmiInputPlugin.md:380]: Change "activates HDMI Input source" to "HDMI Input source is activated".

File: HdmiInput/doc/HdmiInputPlugin.md [hunk 2]
```diff
@@ -380,17 +416,24 @@ Activates the specified HDMI Input port as the primary video source.
...
 <a name="method.stopHdmiInput"></a>
-## *stopHdmiInput <sup>method</sup>*
+## *stopHdmiInput [<sup>method</sup>](#head.Methods)*
 
 Deactivates the HDMI Input port currently selected as the primary video source.
+ 
+### Events 
+| Event | Description | 
+| :----------- | :----------- | 
+| `onInputStatusChanged` | Triggers the event when deactivates HDMI Input source and Input status changes to `stopped`.
```
Comment (anand-ky) [inline HdmiInput/doc/HdmiInputPlugin.md:434]: Change "deactivates HDMI Input source" to "HDMI Input source is deactivated".

**Review by anand-ky — CHANGES_REQUESTED** [review]

File: HdmiInput/doc/HdmiInputPlugin.md (hunk 1 above)
Comment (RKarthikeyanRaja) [reply HdmiInput/doc/HdmiInputPlugin.md]: Updated the changes

Comment (RKarthikeyanRaja) [reply HdmiInput/doc/HdmiInputPlugin.md]: Updated the changes

File: HdmiInput/doc/HdmiInputPlugin.md (hunk 2 above)
Comment (RKarthikeyanRaja) [reply HdmiInput/doc/HdmiInputPlugin.md]: Updated the changes.

**Review by anand-ky — APPROVED** [review]

**Pull request merged**